src/modules/MeasureTool/MeasureToolCore/Measurement.cpp

```diff
@@ -4,89 +4,166 @@
                                             const bool printHeight,
-                                            const Unit units) const
+                                            const int units) const
 {
     PrintResult result;
-    if (printWidth)
-    {
-        result.strLen += swprintf_s(buf,
-                                    bufSize,
-                                    L"%g",
-                                    Width(units));
+
+    auto print = [=, &result](Measurement::Unit unit, const bool paren) {
+        if (paren)
+        {
+            result.strLen += swprintf_s(buf + result.strLen, bufSize - result.strLen, printWidth && printHeight ? L"\n(" : L" (");
+        }
+        if (printWidth)
+        {
+            result.strLen += swprintf_s(buf + result.strLen,
+                                        bufSize - result.strLen,
+                                        L"%.4g",
+                                        Width(unit));
+            if (printHeight)
+            {
+                result.crossSymbolPos[paren] = result.strLen + 1;
+                result.strLen += swprintf_s(buf + result.strLen,
+                                            bufSize - result.strLen,
+                                            L" \x00D7 ");
+            }
+        }
         if (printHeight)
         {
-            result.crossSymbolPos = result.strLen + 1;
             result.strLen += swprintf_s(buf + result.strLen,
                                         bufSize - result.strLen,
-                                        L" \x00D7 ");
+                                        L"%.4g",
+                                        Height(unit));
         }
-    }
-
-    if (printHeight)
-    {
-        result.strLen += swprintf_s(buf + result.strLen,
-                                    bufSize - result.strLen,
-                                    L"%g",
-                                    Height(units));
-    }
-
-    switch (units)
+        switch (unit)
+        {
+        case Measurement::Unit::Pixel:
+            result.strLen += swprintf_s(buf + result.strLen,
+                                        bufSize - result.strLen,
+                                        L" px");
+            break;
+        case Measurement::Unit::Inch:
+            result.strLen += swprintf_s(buf + result.strLen,
+                                        bufSize - result.strLen,
+                                        L" in");
+            break;
+        case Measurement::Unit::Centimetre:
+            result.strLen += swprintf_s(buf + result.strLen,
+                                        bufSize - result.strLen,
+                                        L" cm");
+            break;
+        case Measurement::Unit::Millimetre:
+            result.strLen += swprintf_s(buf + result.strLen,
+                                        bufSize - result.strLen,
+                                        L" mm");
```

Comment:
These strings like `"mm"` are not localizable like this. I think some of these have localizable differences. Can you please make these strings localizable?